### PR TITLE
Piterdaw-Flavour into v.3.0-dev 25.08.2024

### PR DESCRIPTION
**Pull request by piterdaw**

### CODE REVIEW

**Review by MichaHelder — APPROVED** [review]
Mostly tested on Traveler branch. Merging into dev for testing on Unstable.

File: MEIOUandTaxes1/src/common/institutions/00_Core.txt
Comment (Veheled) [inline MEIOUandTaxes1/src/common/institutions/00_Core.txt:1]: Not sure if this is how on_start works.

File: MEIOUandTaxes1/src/events/DG-01-Papacy.txt [hunk 1]
```diff
@@ -6006,7 +6006,7 @@ country_event = {
 	}
 	option = {
 		name = "DG_Papacy.901.opta"
-		add_reform_desire = 0.00002
+		add_reform_desire = 0.00001
```
Comment (Veheled) [inline MEIOUandTaxes1/src/events/DG-01-Papacy.txt:6009]: I am not sure such small values actually do anything. AFAIK most eu4 effects if not all take at max 3 decimal places. This is not an issue with this merge but generally with the DG code

File: MEIOUandTaxes1/src/events/DG-02-Protestant.txt [hunk 2]
```diff
@@ -1,3 +1,10172 @@
...
+			mn_unrest_max_10 =	{ custom_tooltip = mn_unrest_max_3_tt } 
+		}
+	}
+	after = {
+		hidden_effect = {
+			UnrestNofificationCleanup = yes
+		}
+	}
+}
+=======
```
Comment (Veheled) [inline MEIOUandTaxes1/src/events/DG-02-Protestant.txt]: This file is broken due to merge problem. Need to be fixed before merging the pull request.

**Review by Veheled — APPROVED** [review]
Seems fine generally besides a few minor code quality problems and potential stuff not working as intended due to vanilla.

**Pull request merged**